controllers/front/StoresController.php

```diff
@@ -36,7 +36,6 @@ public function init()
     {
         parent::init();
 
-        // StarterTheme: Remove check when google maps v3 is done
         if (!extension_loaded('Dom')) {
```

Comment:
The whole block can be deleted, it's require in composer.json

Reply:
@PierreRambaud But this allows to show an error message during runtime while requiring the `dom` extension in `composer.json` only throws an error when executing `composer install` 🤔

Reply:
@atomiix doesn't matter, we use `SimpleXMLElement` not a `DOMDocument` and PS installer is checking for it 👍